packages/api/src/store/table.ts

```diff
@@ -143,6 +143,25 @@ export default class Table<T extends DBObject> {
     }
   }
 
+  async add(id: string, doc: T) {
```

Comment:
Took me a long time to understand what was happening here. Looks good, but maybe let's add a comment:
```suggestion
  // Takes in an object of {"field": number} and increases all the fields by the specified amounts
  async add(id: string, doc: T) {
```